lib/ansiblelint/rules/MissingFilePermissionsRule.py

```diff
@@ -19,57 +19,77 @@
 # THE SOFTWARE.
 from ansiblelint.rules import AnsibleLintRule
 
+# Despite documentation mentioning 'preserve' only these modules support it:
+_modules_with_preserve = (
+    'copy',
+    'template',
+)
+
 
 class MissingFilePermissionsRule(AnsibleLintRule):
     id = "208"
-    shortdesc = 'File permissions not mentioned'
+    shortdesc = 'File permissions not mentioned or incorrect'
     description = (
-        "Missing mode parameter can cause unexpected file permissions based "
-        "on version of Ansible being used. Be explicit, or if you still "
-        "want the default behavior you can use ``mode: preserve`` to avoid "
-        "hitting this rule. See "
-        "https://github.com/ansible/ansible/issues/71200"
+        f"Missing or unsupported mode parameter can cause unexpected file "
```

Comment:
f-prefix is only necessary on lines with interpolation
```suggestion
        "Missing or unsupported mode parameter can cause unexpected file "
```